src/core/flow.js

```diff
@@ -373,13 +385,17 @@ export function FlowWithPhases({
   };
 
   const startTurn = function(state, config) {
-    const G = config.onTurnBegin(state.G, state.ctx);
+    if (config.secondsPerTurn) {
+      setTimeout(() => {
+        endTurnEvent(state);
```

Comment:
On line 374 it reads `state = endPhaseEvent(state);`
But here on line 390, it should be doing a similar thing but it reads
`endTurnEvent(state);`

So one of these is different than the other (one is assigning to the `state` variable and one isn't), this could be causing problems?